src/form-elements/FormElementRadio.tsx

```diff
@@ -122,4 +107,41 @@ function FormElementRadio({
   )
 }
 
+type RadioOptionButtonProps = {
+  element: Props['element']
+  option: FormTypes.ChoiceElementOption
+  isSelected: boolean
+  onClick: () => void
+}
+const RadioOptionButton = React.memo(function RadioOptionButton({
```

Comment:
the only difference i can see between this and `CheckboxOptionButton ` is the class names, is it worth making this able to accept those as props? 

eg 

```
{filteredOptions.map((option) => (  <OptionsButton {...props} className="ob-button-radio-container" buttonClassName="ob-radio__button cypress-radio-button-control" /> )
```

Reply:
yep sounds like a good idea